pywps/inout/basic.py

```diff
@@ -310,18 +325,22 @@ def set_data(self, data):
 class BasicIO:
     """Basic Input/Output class
     """
-    def __init__(self, identifier, title=None, abstract=None, keywords=None):
+    def __init__(self, identifier, title=None, abstract=None, keywords=None,
+                 min_occurs=1, max_occurs=1, metadata=[]):
         self.identifier = identifier
         self.title = title
         self.abstract = abstract
         self.keywords = keywords
+        self.min_occurs = int(min_occurs)
+        self.max_occurs = int(max_occurs)
+        self.metadata = metadata
 
 
 class BasicLiteral:
     """Basic literal Input/Output class
     """
 
-    def __init__(self, data_type="integer", uoms=None):
+    def __init__(self, data_type="integer", uoms=[UOM("metre")]):
```

Comment:
What's the rationale for setting meter as the default unit ? I think no unit would be preferable.

Reply:
My bad, there is no reason for default (mandatory) uom. Removed